lib/benchmark_driver/runner/ips.rb

```diff
@@ -107,8 +107,8 @@ def run_benchmark(job, context:)
 
     duration = Tempfile.open(['benchmark_driver-', '.rb']) do |f|
       with_script(benchmark.render(result: f.path)) do |path|
-        IO.popen([*context.executable.command, path], &:read) # TODO: print stdout if verbose=2
-        if $?.success? && ((value = Float(f.read)) > 0)
+        success = execute(*context.executable.command, path, exception: false)
```

Comment:
I don't think `exception: false` is related to showing command output, and it's intentional we raise it.

Reply:
Sorry never mind... we haven't raised it here.

Reply:
Because of some other libraries I tend to use the term `execute` when I assert command is successful. So I got confused, but to share the command output here, it's understandable.